src/sentry/conf/types/consumer_definition.py

```diff
@@ -23,3 +23,7 @@ class ConsumerDefinition(TypedDict, total=False):
     require_synchronization: bool
     synchronize_commit_group_default: str
     synchronize_commit_log_topic_default: str
+
+    dlq_topic: str
+    dlq_max_invalid_ratio: float
+    dlq_max_consecutive_count: int
```

Comment:
Shouldn't these be optional? How is this even passing type checks?

Reply:
Yeah, it's just a bit more explicit to set to None and you can pass the value straight to arroyo

Reply:
Done, made them optional